index.html

```diff
@@ -10,22 +10,23 @@
         <title>Friday</title>
     </head>
     <body>
-        <header>
-            <div class="header-container">
+        <header id="header-bg">
+            <div class="header-container" id="header">
                 <nav class="navigation">
                     <ul>
-                        <li><a><img src="images/image_logo.png">Friday</a></li>
-                        <li><a>About</a></li>
-                        <li><a>Contact</a></li>
-                        <li><a><img src="#"></a></li>
+                        <!-- Need help with alignment, please. -->
+                        <li><a><img src="images/image_logo.png" alt="Friday Logo" id="logo"></a></li>
```

Comment:
basically, we should have all resources such as images, other css files managed under the folder resources. Could you move that?